src/TableRow.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Moment from "react-moment";
+import { useState } from "react";
+
+const TableRow = ({
+  id,
+  title,
+  firstName,
+  surname,
+  email,
+  roomId,
+  checkInDate,
+  checkOutDate,
+  showProfile
+}) => {
+  const [color, setColor] = useState(false);
+
+  function getProfile() {
```

Comment:
This is not essential, but when you click the "Show Profile" button, the row also gets highlighted. That could be confusing to users.

This happens because the click event gets handled by `button`, but then also bubbles up to the `tr` that the `button` is inside, and the `onClick` on `tr` highlights the row.

If you wanted the "Show Profile" button to just display the profile `div`, without  also highlighting the row, you could call `stopPropagation()`, so the event is handled only by `button` and then React prevents it reaching parent components. 

Like this:

```
function getProfile(e) {
   e.stopPropagation();
   //... rest of your code
```

For more info, see: https://reactjs.org/docs/events.html#overview